ci/dcos.sc

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env amm
+
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+import scala.io.Source
+import scala.util.Try
+
+import $file.utils
+import utils.SemVer
+
+// output directory for all packages going to S3
+val PACKAGE_DIR: Path = pwd / 'target / 'universal
+// dcos catalog template directory
+val TEMPLATE_DIR: Path = pwd / 'dcos
+// versioned dcos catalog directory
+val UNIVERSE_DIR: Path = pwd / 'target / 'dcos
+
+@main
+def localBuild(version: String) {
+  buildRegistry(SemVer(version, ""))
+}
+
+/**
+ * Main function for building DCOS registry file that works
+ * with DCOS 1.12+.
+ *
+ * @param version updated in the DCOS package files
+ */
+
+def buildRegistry(version: SemVer) {
```

Comment:
procedural syntax deprecated: https://contributors.scala-lang.org/t/proposal-to-remove-procedure-syntax/2143

Please use `: Unit  = {`

Reply:
roger that.. thx!